subgraphs/erc721/schema.graphql

```diff
@@ -0,0 +1,199 @@
+###
+# Subgraph Schema: ERC721 Smart Contracts and Tokens
+# Version: 1.0.0
+
+#############################
+####### Token Metadata ######
+#############################
+
+type Token @entity {
+  " address of the token, { Contract Address }-{ Token ID }  "
+  id: ID!
+
+  " Smart contract address for the token "
+  contract: Contract!
+
+  " ID of the token from the smart contract "
+  tokenID: BigInt!
+
+  " URI of the token "
+  tokenURI: String!
+
+  " The owner's address for the token "
+  owner: Account!
+
+  " timestamp of the token's mint "
+  mintTime: BigInt!
+}
+
+#############################################
+####### ERC721 Smart Contract Metadata ######
+#############################################
+
+type Contract @entity {
+  " ERC721 Smart contract address "
+  id: ID!
+
+  " Name of the smart contract "
+  name: String!
+
+  " Symbol of the smart contract "
+  symbol: String!
+
+  " Flag for whether EIP721 metadata is supported "
+  supportsEIP721Metadata: Boolean!
+
+  " Total number of unique tokens in the smart contract "
+  tokenCount: BigInt!
```

Comment:
do you mean minted token count?

Reply:
Yes. Update the comment part accordingly. Please see an updated version at https://github.com/messari/subgraphs/pull/689/commits